### PR TITLE
Chore/tests

### PR DESCRIPTION
**Pull request by BeccaSheldon**
Lots of test updates; section at the bottom has problem/unfinished areas that I'll update tonight/ tomorrow, unless you guys think they aren't necessary. @mrgordon @arvel @maj @syrnick

### CODE REVIEW

File: test/integration_tests.rb [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 
-require 'rubygems'
 require 'crowdflower'
 require 'json'
 
-API_KEY   = ENV["API_KEY"]
-DOMAIN_BASE = ENV["DOMAIN_BASE"] || "https://api.localdev.crowdflower.com:8443"
+API_KEY = YAML::load(File.read("api.yml")) 
+#add PN to yml file also 
+DOMAIN_BASE = "https://api.crowdflower.com" || "https://api.localdev.crowdflower.com:8443"
```
Comment (mrgordon) [inline test/integration_tests.rb:9]: this looks like a local hack. the first URL will always be returned. not sure re: PN yml file comment

File: test/integration_tests.rb [hunk 2]
```diff
@@ -57,116 +54,289 @@ def assert_exception_raised expected_exception_class
...
+say "Updating job_3 title."
+job_3.update(:title => 'Job_3: Copy of Job_2')
+assert job_3.get["title"] == "Job_3: Copy of Job_2"
+
+say "-- Waiting for CrowdFlower to process the data."
+wait_until { job_3.get["units_count"] == 13 }
+assert job_3.get["units_count"] == 13
+
+#################################################
+# WOKER METHODS - missing assetions
```
Comment (mrgordon) [inline test/integration_tests.rb]: some typos

File: test/integration_tests.rb [hunk 3]
```diff
@@ -57,116 +54,289 @@ def assert_exception_raised expected_exception_class
...
+
+say "Flagging worker from all my jobs."
+worker.flag(worker_id, "Testing flag method across all jobs.", :persist => true)
+
+say "Deflagging worker."
+worker.deflag(worker_id, "Testing deflag method.") 
+
+# Be careful if testing the reject method; cannot be undone
+# say "Rejecting worker."
+# worker.reject(worker_id)
```
Comment (mrgordon) [inline test/integration_tests.rb]: go ahead and remove this if we're gutting the method

Comment (BeccaSheldon) [inline test/integration_tests.rb]: wait, are we removing worker.reject? I thought only judgment.reject.

Comment (mrgordon) [inline test/integration_tests.rb]: we can leave that in. basic users cannot reject if I recall correctly but other users will find it useful

**Comment by BeccaSheldon** [comment]
@arvel the disable_channel method is added with tests and readme updates. The tests went haywire for some reason, but everything should run smoothly now if you give them a try.

**Comment by arvel** [comment]
+1

**Pull request merged**